ert_shared/share/ert/shell_scripts/careful_copy_file

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
-import sys, os
+from asyncio.log import logger
```

Comment:
Is this needed?

Reply:
nope, vscode is a bit overzealous at times :see_no_evil: